### ISSUE TITLE
jump to page feature

### ISSUE THREAD

**Issue by joeyspaaa2** [filipdanic/spicy-datatable#10]
When you have a lot of pages, the three dots appear. Can you include a jump to page feature there? 

E.g.:

Click on the 3 dots and a small tooltip with input field appears. This would be great.

**Comment by filipdanic** [comment]
Yeah, I’ve seen this pattern used with many pagination-based systems. Never really used it, I don’t really feel there is a major use-case here?

I’m all for this being added if someone is willing to work on it of course. It can be an optional addition to library.
